.github/workflows/hourly-run.yml

```diff
@@ -0,0 +1,105 @@
+on:
+  schedule:
+    - cron:  '30 * * * *' # every hour at `x:30`
+  workflow_dispatch:
+
+name: Hourly check
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  self-care:
+    name: Flake self-check
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Check Nix flake inputs
+        uses: DeterminateSystems/flake-checker-action@v5
+        with:
+          fail-mode: true
+          nixpkgs-keys: nixpkgs
+
+  test:
+    if: github.repository == 'fedimint/fedimint'
+    strategy:
+      matrix:
+        host:
+          - linux
+        include:
+          - host: linux
+            runs-on: buildjet-8vcpu-ubuntu-2004
```

Comment:
[nit] Should we consider bumping our buildjet runners from 2004 to 2204?

Reply:
Ubuntu 20.04 :D